package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "nodejs-mp",
+  "version": "1.0.0",
+  "description": "NodeJS mentoring program hometasks",
+  "main": "index.js",
+  "dependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-preset-env": "^1.6.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
+    "babel-register": "^6.26.0",
+    "babel-watch": "^2.0.7"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "start": "babel-watch index.js",
+    "test": "npm run test"
```

Comment:
This is recursive invalid script. Let's replace it with commented call of some testing tool (e.g. "jest")